lib/absinthe/type/built_ins/scalars.ex

```diff
@@ -102,13 +102,10 @@ defmodule Absinthe.Type.BuiltIns.Scalars do
   end
 
   @spec parse_string(any) :: {:ok, binary} | :error
-  defp parse_string(value) do
-    try do
-      {:ok, to_string(value)}
-    rescue
-      Protocol.UndefinedError -> :error
-    end
+  defp parse_string(value) when is_binary(value) do
+    {:ok, value}
   end
+  defp parse_string(_), do: :error
```

Comment:
@bruce I think this is a better route for parsing strings. `Kernel.to_string` is entirely too forgiving with inputs. For example it'll take a list of integers `[1,2,3]` and turn it into `<<1,2,3>>` which is definitely not gonna be considered normal.

I could see an argument for coercing floats / integers to binaries, but I think that ought to be the limit.